examples/api/collections_demo.py

```diff
@@ -38,9 +38,9 @@
 yo = rs.randn(npts)
 xyo = list(zip(xo, yo))
 
-# Make a list of colors cycling through the rgbcmyk series.
-colors = [colorConverter.to_rgba(c)
-          for c in ('r', 'g', 'b', 'c', 'y', 'm', 'k')]
+# Make a list of colors cycling through the default series.
+colors = [colorConverter.to_rgba(c['color'])
+          for c in list(plt.rcParams['axes.prop_cycle'])]
```

Comment:
Really need to release next version of cycler so we can use https://github.com/matplotlib/cycler/blob/master/cycler.py#L349